### ISSUE TITLE
Keyboard access to expandable sidebar menu items is non-standard

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#847]
**Describe the bug**
Keyboard access to expandable sidebar menu items is non-standard and will frustrate keyboard user

**To Reproduce**
1. https://openfl.readthedocs.io/en/latest/index.html 
2. Use keyboard TAB key to navigate between sidebar items
3. Use ENTER to expand/collapse items

Example: As a keyboard user I can’t get to ‘core components’ under ‘openfl structure’ without first activating the OpenFL Structure link, which loads the overall topic page for that link. 

Notice:

1. The expand/collapse controls should be always visible and always in focus order
2. Since the top level of each expandable section is also itself a link to a landing page, the keyboard user can’t expand a section without activating the link to go to the overall landing page
4. Usability-wise, it also doesn’t visually distinguish which items are collapsible sections (the first two, manual and openfl structure) from the items that are links (openfl troubleshooting; notices and disclaimers) 
5. Further, focus is not consistently returned to / moving to where it should; sometimes I can get to next sub-link as expected, sometimes next focusable item is one of the next links. 

Expect:
1. The expand/collapse controls should be always visible and always in focus order
2. Do not have multiple actions triggered on activation of a link; separate the link to landing page for a section from the expand/collapse behavior
3. Ensure that focus moves consistently through all available links without skipping links / sections

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.

**Comment by fstrr** [comment]
These are issues with the [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).

**Comment by theakshaypant** [comment]
Closing. As mentioned by @fstrr, this is an issue with [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).
